doc/introduction/data.rst

```diff
@@ -39,7 +31,7 @@ The :func:`~pennylane.data.load` function returns a ``list`` with the desired da
 >>> H2data = H2datasets[0]
 
 We can load datasets for multiple parameter values by providing a list of values instead of a single value.
-To load all possible values, use the special keyword "full".
+To load all possible values, use the special value :const:`~pennylane.data.FULL`.
 
 >>> H2datasets = qml.data.load("qchem", molname="H2", basis="full", bondlength=[0.5, 1.1])
```

Comment:
If we are doing the above, then I guess all examples need to change as well. Although, I would instead prefer letting users use `"full"` and using `data.FULL` internally.

Reply:
Okay, we'll stick to `"full"` in the docs